docs/en/sql-reference/functions/string-functions.md

```diff
@@ -2532,3 +2532,37 @@ Result:
 │ foo                        │
 └────────────────────────────┘
 ```
+## strncmp
+
+Compare two strings.
+
+**Syntax**
+```sql
+strncmp(a_str, a_offset, b_str, b_offset, n);
+```
+
+**Arguments**
+- `a_str` — The first string to compare.
+- `a_offset` — The starting position (zero-based index) in `a_str` from which the comparison begins.
+- `b_str` — The second string to compare.
+- `b_offset` — The starting position (zero-based index) in `b_str` from which the comparison begins.
+- `n` — The number of bytes to compare in both strings, starting from their respective offsets.
```

Comment:
For the arguments (l. 2545ff), we'll need to document/mention also their allowed datatypes.